external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -3807,11 +3807,10 @@ fn assert_use_def_with_doc_string(
         "for use in column {use_col} of line {use_line} in file {use_file}"
     );
     let info = def_info.get(&use_def.def_loc).unwrap();
+    let info_str = format!("{}", info);
```

Comment:
Nit
```suggestion
    let info_str = info.to_string();
```

Reply:
I'm not sure it implements to_string? Or does anything that has Display has to_string?

Reply:
Correct -- anything that is `Display` has `to_string` -- `to_string` calls `Display` underneath it https://doc.rust-lang.org/std/string/trait.ToString.html